screwdriver.yaml

```diff
@@ -0,0 +1,44 @@
+---
+workflow:
+    - beta
+
+shared:
+    image: node:6
+
+jobs:
+    main:
+        steps:
+            - check_build_requirements: |
+                which curl
+                which grep
+    beta:
+        steps:
+            - check_parameters: |
+                if [ -z "$K8S_TOKEN" ] || [ -z "$K8S_TAG" ] || [ -z "$K8S_DEPLOYMENT" ] || [ -z "$K8S_CONTAINER" ] || [ -z "$K8S_IMAGE" ] || [ -z "$K8S_HOST" ]; then
+                  echo Unable to kubernetes trigger, missing environment variables
+                  exit 2
+                fi
+            - deploy_k8s: |
```

Comment:
It's different. It does not update a `deployments` resource, it updates a `statefulsets` resource